doc/how-to-use.md

```diff
@@ -0,0 +1,400 @@
+## How to use multus-cni?
+
+### Prerequisite
+
+Kubelet must be configured to run with the CNI network plugin. Edit `/etc/kubernetes/kubelet` file and add `--network-plugin=cni` flags in `KUBELET\_OPTS `as shown below:
```

Comment:
`KUBELET\_OPTS ` should be `KUBELET_OPTS`

Reply:
Removed 'KUBELET_OPTS'.